Curtains/Curtains.Domain/Models/ReviewModel.cs

```diff
@@ -31,7 +31,7 @@ public class ReviewModel : BaseEntity
         /// <summary>
         ///  This property represents path to image of review.
         /// </summary>
-        public string Image { get; set; }  
+        public string ImagePath { get; set; }  
```

Comment:
I think it's not difficult to rename it in each model :)

Reply:
We're use "ImagePath" in every model